[2022-05-13 17:01:29,786] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-13T11:31:11.197884+00:00 [queued]>
[2022-05-13 17:01:29,800] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-13T11:31:11.197884+00:00 [queued]>
[2022-05-13 17:01:29,800] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 17:01:29,800] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-13 17:01:29,800] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 17:01:29,834] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-05-13 11:31:11.197884+00:00
[2022-05-13 17:01:29,835] {standard_task_runner.py:52} INFO - Started process 8977 to run task
[2022-05-13 17:01:29,839] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-05-13T11:31:11.197884+00:00', '--job-id', '40', '--raw', '--subdir', '/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmprfh3362o', '--error-file', '/tmp/tmppf7x5ktg']
[2022-05-13 17:01:29,840] {standard_task_runner.py:80} INFO - Job 40: Subtask run_after_loop
[2022-05-13 17:01:29,902] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-05-13T11:31:11.197884+00:00 [running]> on host roshan-Latitude-E5440
[2022-05-13 17:01:29,978] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T11:31:11.197884+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-13T11:31:11.197884+00:00
[2022-05-13 17:01:29,979] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-13 17:01:29,981] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-05-13 17:01:29,988] {subprocess.py:85} INFO - Output:
[2022-05-13 17:01:29,989] {subprocess.py:92} INFO - 1
[2022-05-13 17:01:29,989] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-13 17:01:30,029] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220513T113111, start_date=20220513T113129, end_date=20220513T113130
[2022-05-13 17:01:30,055] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-13 17:01:30,084] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
